YuliiaBaz/PTMP
# Simple script

Prerequisites:

Manually test site’s workability

Description:

Create simple script to open Home page, open Contacts.

Number of users: up to 10

User role: anonymous

Goals:

Check availability of the site from the load tool

Make sure the site is configured for multiuser mode

Setup basic metrics and make sure they can be collected

Setup ramp-up mode in the scenario
